tests/mappings/manifest_test_lib.py

```diff
@@ -38,14 +38,17 @@ def assertManifestsMatch(self, expected_path, got_path):
     with open(g_file, mode='rt', encoding='utf-8') as g_fp:
       got = json.loads(g_fp.read())
     got_dict = {x['dest']: x for x in got}
-    # self.assertEqual(expected_dict, got_dict)
 
     ok = True
     expected_dests = set(expected_dict.keys())
     got_dests = set(got_dict.keys())
     for dest, what in expected_dict.items():
       got = got_dict.get(dest)
       if got:
+        # bazlmod mode changes root to @@//, but history says @//
```

Comment:
1. nit -- "Bzlmod" instead of "bazlmod"
2. just for my information, what does "history says @//" mean?

Reply:
changed